safeeyes/safeeyes/config/lang/pt.json

```diff
@@ -0,0 +1,41 @@
+{
+    "meta_info": {
+        "language_name": "Português",
+        "language_name_en": "Portuguese"
+    },
+    "exercises": {
+        "long_break_exercises": [
+            "Caminhe um pouco",
+            "Encoste na cadeira e relaxe"
+        ],
+        "short_break_exercises": [
+            "Feche bem os olhos",
+            "Mexa oscd  olhos",
```

Comment:
Hi,
Thank you for making Safe Eyes better :-).
Could you please make sure whether this line is correct? I don't know anything about Portuguese but Google translate does not correctly translate `Mexa oscd  olhos` and suggests `Mexe os olhos`  instead. I know Google translate is not perfect but to avoid human error I double check the translations using Google. If there is nothing to change in your translation, please comment below and I will merge your contribution. Sorry for the trouble and thanks again :-)